Assets/UniSense/LowLevel/DualSenseHIDInputReport.cs

```diff
@@ -92,6 +92,30 @@ internal struct DualSenseHIDInputReport : IInputStateTypeInfo
         [InputControl(name = "micMuteButton", layout = "Button", displayName = "Mic Mute", bit = 2)]
         [FieldOffset(10)]
         public byte buttons3;
+        
+        [InputControl(name = "gyroPitch", layout = "Axis", format = "SHRT")]
+        [FieldOffset(16)] 
+        public short gyroPitch;
+        
+        [InputControl(name = "gyroYaw", layout = "Axis", format = "SHRT")]
+        [FieldOffset(18)] 
+        public short gyroYaw;
+        
+        [InputControl(name = "gyroRoll", layout = "Axis", format = "SHRT")]
+        [FieldOffset(20)] 
+        public short gyroRoll;
+    
+        [InputControl(name = "accelX", layout = "Axis", format = "SHRT")]
+        [FieldOffset(22)] 
+        public short accelX;
+        
+        [InputControl(name = "accelY", layout = "Axis", format = "SHRT")]
+        [FieldOffset(24)] 
+        public short accelY;
+        
+        [InputControl(name = "accelZ", layout = "Axis", format = "SHRT")]
+        [FieldOffset(26)] 
+        public short accelZ;
```

Comment:
After I saw this PR, I tried to change your code to use VC3S.
I think it works properly in my environment.
How about this:

```suggestion
        [InputControl(name = "gyro", format = "VC2S", layout = "Vector3")]
        [InputControl(name = "gyro/x", layout = "Axis", format = "SHRT")]
        [InputControl(name = "gyro/y", offset = 2, layout = "Axis", format = "SHRT")]
        [InputControl(name = "gyro/z", offset = 4, layout = "Axis", format = "SHRT")]
        [FieldOffset(16)] public short gyroPitch;
        [FieldOffset(18)] public short gyroYaw;
        [FieldOffset(20)] public short gyroRoll;
    
        [InputControl(name = "accel", format = "VC2S", layout = "Vector3")]
        [InputControl(name = "accel/x", layout = "Axis", format = "SHRT")]
        [InputControl(name = "accel/y", offset = 2, layout = "Axis", format = "SHRT")]
        [InputControl(name = "accel/z", offset = 4, layout = "Axis", format = "SHRT")]
        [FieldOffset(22)] public short accelX;
        [FieldOffset(24)] public short accelY;
        [FieldOffset(26)] public short accelZ;
```